MWT/how-to-container.rst

```diff
@@ -14,8 +14,7 @@ Computing the optimal size of the container
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The ``computeContentOptimalSize()`` method is called by the MWT framework in order to know the optimal size of the container.
-The optimal size of the container should be big enough so that each child can be allocated a size at least as big as its own optimal size.
-For example, the optimal width of a horizontal list is the sum of the optimal width of every child, and the optimal height is the maximum optimal height of all children.
```

Comment:
Why did you remove this example?

Reply:
instead i added a code snippet to compute the optimal size of a container. i chose a simple wrapper rather than a horizontal list to have a shorter example

Reply:
ok